src/vehicle/const.py

```diff
@@ -41,6 +41,7 @@ class VehicleInterior(str, Enum):
     SEDAN = "sedan"
     STATION_WAGON = "stationwagen"
     PICK_UP_TRUCK = "pick-up truck"
+    NOT_AVAILABLE = "N.v.t."
```

Comment:
This is unreachable? `N.v.t.` is filtered by the model

https://github.com/frenck/python-vehicle/blob/main/src/vehicle/models.py#L149

Reply:
Good catch. I've been over productive ;). Const.py can indeed be left unchanged. Sorry.

Reply:
I can't seem to delete the const.py file from the pull-request via the site; I'll have to figure out the CLI now. Please bear with me :)

Reply:
Ok, just removed the added line in const.py.